recipe/meta.yaml

```diff
@@ -12,27 +12,30 @@ source:
 build:
   entry_points:
     - esbonio = esbonio.__main__:main
-    - esbonio-sphinx = esbonio.lsp.sphinx.cli:main
+    - esbonio-sphinx = esbonio.sphinx_agent.__main__:main
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
   number: 0
 
 requirements:
   host:
     - pip
-    - setuptools
     - python {{ python_min }}
+    - hatchling
   run:
-    - platformdirs
-    - pygls >=1.1.0
-    - pyspellchecker
     - python >={{ python_min }}
+    - aiosqlite
+    - platformdirs
+    - docutils
+    - pygls >=2.0
+    - tomli
+    - websockets
     - sphinx
-    - typing-extensions
 
 test:
   imports:
-    - esbonio.lsp
+    - esbonio.server
+    - esbonio.sphinx_agent
```

Comment:
`esbonio/sphinx_agent/__init__.py` has no code in it, could you change this to be `esbonio.sphinx_agent.types` instead?